### PR TITLE
Added the dom purify for the message

### PR DESCRIPTION
**Pull request by panwaranuj01**
### What do you want to achieve?
- Added the dom purify for the message

### PR Checklist

- [x] I have tested it locally and all functionalities are working fine.
- [x] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

-

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

### Screenshot
- 

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: webplugin/lib/js/dom-purify-3.1.4.min.js [hunk 1]
```diff
@@ -0,0 +1,684 @@
+/*! @license DOMPurify 3.1.4 | (c) Cure53 and other contributors | Released under the Apache license 2.0 and Mozilla Public License 2.0 | github.com/cure53/DOMPurify/blob/3.1.4/LICENSE */
```
Comment (devashishmamgain) [inline webplugin/lib/js/dom-purify-3.1.4.min.js:1]: this file doesn't look like a minified version, please recheck

File: webplugin/js/app/mck-sidebox-1.0.js [hunk 2]
```diff
@@ -9378,6 +9378,11 @@ var userOverride = {
                 var attachmentBox = 'n-vis';
                 var kmAttchMsg = '';
                 let isUserMsg = true;
+
+                if (msg?.message) {
+                    msg.message = window.DOMPurify.sanitize(msg.message);
```
Comment (devashishmamgain) [inline webplugin/js/app/mck-sidebox-1.0.js:9383]: what is the purpose of appending window. here
is it because of iFrame thing?

**Review by devashishmamgain — CHANGES_REQUESTED** [review]

File: webplugin/js/app/mck-sidebox-1.0.js (hunk 2 above)
Comment (panwaranuj01) [reply webplugin/js/app/mck-sidebox-1.0.js:9383]: window is referring to the iframe context here. 
This is a similar `window.Dom || Dom` but following the widget code convention

File: webplugin/lib/js/dom-purify-3.1.4.min.js (hunk 1 above)
Comment (panwaranuj01) [reply webplugin/lib/js/dom-purify-3.1.4.min.js:1]: ok, it looks like I have done the formatting here  😄
will revert this

**Review by devashishmamgain — APPROVED** [review]

**Review by devashishmamgain — APPROVED** [review]

**Pull request merged**